senders/victorops/send.go

```diff
@@ -0,0 +1,112 @@
+package victorops
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/writeas/go-strip-markdown"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/senders/victorops/api"
+)
+
+// SendEvents implements Sender interface Send
+func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plot []byte, throttled bool) error {
+	var messageType api.MessageType
+	switch events[len(events)-1].State {
+	case moira.StateERROR:
+		messageType = api.Critical
+	case moira.StateEXCEPTION:
+		messageType = api.Critical
+	case moira.StateWARN:
+		messageType = api.Warning
+	case moira.StateOK:
+		messageType = api.Recovery
+	case moira.StateNODATA:
```

Comment:
For us, the NODATA state is as bad as the error state, let's use here api.Critical